.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+
+updates:
+
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
```

Comment:
@reitermarkus Can we move these back to being immediate updates? Weekly is a long time to wait for security updates.

Also: does this support more than security updates e.g. patch versions etc.?

Reply:
> Can we move these back to being immediate updates? 

The immediate/daily updates are way too noisy if you ask me.

> Also: does this support more than security updates e.g. patch versions etc.?

What do you mean exactly?

Reply:
> The immediate/daily updates are way too noisy if you ask me.

If we can selectively make some (noisy) gems not be immediate: that'd be fine. We want to have immediate security updates.

> What do you mean exactly?

Prevoiously we had stuck with the "old" dependabot because the new one didn't support non-security bumps, initially at least. Has this changed?

Reply:
> If we can selectively make some (noisy) gems not be immediate: that'd be fine. We want to have immediate security updates.

Which of our dependencies actually pose a security threat that requires immediate action?

> Has this changed?

It just bumped the `sorbet` Gems and it's always bumping the GitHub actions in cask repos, so I think so.

Reply:
> Which of our dependencies actually pose a security threat that requires immediate action?

Potentially: any of those that are vendored.